pkg/log/log.go

```diff
@@ -150,6 +151,29 @@ func format(msg, level string, ts time.Time, appInfo Marshaler, mm Many) string
 		generateFatalFields(entry)
 	}
 
+	// Attempt to map the caller of the log function into the "source" field for identifying if a service or a module
```

Comment:
nit: I'd suggest pulling this out into its own function so we reduce the bloat of this function. That being said, I don't really care about this logger at all and want to kill it with fire as soon as possible so I am happy to ignore this

Reply:
ya totes fair